ts/lib/auth/computeclient.ts

```diff
@@ -16,8 +16,18 @@
 
 import * as request from 'request';
 
+import {RequestCallback, RequestError} from './../transporters';
 import Auth2Client from './oauth2client';
 
+export interface Tokens {
+  expires_in: number;
+  expiry_date: number;
+}
+
+export interface RefreshTokenCallback {
```

Comment:
Why did you choose to use an interface here?  If you wanted to, I think you could also use something like the following.
```
export declare type RefreshTokenCallback = (err: Error, tokens: Tokens, response: request.RequestReponse) => void;
```